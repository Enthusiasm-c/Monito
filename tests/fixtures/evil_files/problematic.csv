Название,Цена,Категория,Описание
iPhone 14,"99,999",Смартфоны,Новый iPhone с камерой
,89999,Смартфоны,Samsung Galaxy
MacBook Pro,дорого,Ноутбуки,
iPad Air,59999.99,,Планшет Apple
"AirPods""Pro",25000,Аксессуары,"Наушники
с шумоподавлением"
Товар #123,0,Тестовая,Спецсимволы: @#$%^&*()
