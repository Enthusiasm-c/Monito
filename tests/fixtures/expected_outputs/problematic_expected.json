{
  "filename": "problematic.csv",
  "expected_rows_in": 6,
  "expected_rows_out": 4,
  "challenges": [
    "Empty cells",
    "Non-numeric prices",
    "Special characters",
    "Multiline text"
  ]
}